### DISCUSSION TITLE
use ldap3.NONE for parameter get_info in ldap_auth function to accelerate login

### DISCUSSION THREAD

**Discussion by lich72** [open-webui/open-webui#8204] [Ideas]
# Feature Request

**Is your feature request related to a problem? Please describe.**

The project is a wonderful project and helps a lot.

We have deployed it in our internal network. But after our test, there is a little problem which brothering us that it takes more than 8 seconds to login in our environment.

**Describe the solution you'd like**

I searched related information and found there should be a solution. We can replace a value of parameter to solve.

https://github.com/open-webui/open-webui/blob/2299f4843003759290cc6bf823595c6578ee4470/backend/open_webui/routers/auths.py#L204

from "ALL" to "ldap3.NONE"

**Describe alternatives you've considered**

**Additional context**

Refer to [https://github.com/cannatag/ldap3/issues/534](https://github.com/cannatag/ldap3/issues/534), use NONE will obviously accelerate LDAP search.

**Comment by steelcg** [comment]
It's implemented in PR: #8376 
Please close it.
